File: Estrutura de Dados Simples/Apresentacao_Nomes.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

             String[] matrizA = new String[10];

        System.out.println("--- Digite 10 nomes ---");
        for (int i = 0; i < 10; i++) {
            System.out.print("Digite o " + (i + 1) + "º nome: ");
            matrizA[i] = scanner.nextLine();
        }

        System.out.println("\n--- Nomes Inseridos ---");
        for (String nome : matrizA) {
            System.out.println(nome);
        }
    }
}

File: Estrutura de Dados Simples/Apresentacao_Nomes.cpp
#include <iostream>
#include <vector>
#include <string>

int main() {
    std::vector<std::string> matrizA(10);

    std::cout << "--- Digite 10 nomes ---" << std::endl;
    for (int i = 0; i < 10; i++) {
        std::cout << "Digite o " << i + 1 << "º nome: ";
        std::getline(std::cin, matrizA[i]);
    }

    std::cout << "\n--- Nomes Inseridos ---" << std::endl;
    for (const auto& nome : matrizA) {
        std::cout << nome << std::endl;
    }

    return 0;
}